mingw-w64-strumpack/PKGBUILD

```
@@ -0,0 +1,43 @@
+# Maintainer: Rafał Brzegowy <rafal.brzegowy@yahoo.com>
+
+_realname=strumpack
+pkgbase=mingw-w64-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=7.2.0
+pkgrel=1
+pkgdesc='Structured Matrix Package (mingw-w64)'
+arch=('any')
+mingw_arch=('mingw64' 'ucrt64' 'clang64')
+depends=("${MINGW_PACKAGE_PREFIX}-openblas"
+         "${MINGW_PACKAGE_PREFIX}-metis"
+         "${MINGW_PACKAGE_PREFIX}-scotch"
+	 "${MINGW_PACKAGE_PREFIX}-omp")
```

```suggestion
         "${MINGW_PACKAGE_PREFIX}-omp")
```
Please use spaces instead of tabs